pyproject.toml

```diff
@@ -47,7 +47,11 @@ redis = { version = ">=4.4,<6.0", optional = true }
 typing-extensions = "^4.11.0"
 datadog-lambda = { version = ">=4.77,<6.0", optional = true }
 aws-encryption-sdk = { version = "^3.1.1", optional = true }
+ddtrace = "^2.5.2"
+opentelemetry-api = "^1.22.0"
+opentelemetry-sdk = "^1.22.0"
 jsonpath-ng = { version = "^1.6.0", optional = true }
+types-aws-xray-sdk = "^2.13.0.20240308"
```

Comment:
Probably needs to be moved to a dev dependency.

Reply:
remove before merge